mbr.go

```diff
@@ -151,6 +151,34 @@ func (this MBR) IsGPT() bool {
 	return false
 }
 
+// MakeProtective - Make this MBR a GPT Protective MBR
+//   sectorSize is either 512 or 4096. diskSize is the size of entire disk in bytes.
+//   https://en.wikipedia.org/wiki/GUID_Partition_Table#Protective_MBR_(LBA_0)
+func (this *MBR) MakeProtective(sectorSize int, diskSize uint64) {
+
+	this.FixSignature()
+
+	// create one partition that spans the whole addressable disk.
+	ptLenLBA := uint32(0xFFFFFFFF)
+	if diskSize/uint64(sectorSize) <= uint64(0xFFFFFFFF) {
+		ptLenLBA = uint32(diskSize/uint64(sectorSize)) - 1
+	}
+	pt := this.GetPartition(1)
+	pt.SetType(PART_GPT)
+	pt.SetLBAStart(1)
+	pt.SetLBALen(ptLenLBA)
```

Comment:
As I read in wikipedia -  max  address must be in uint32, else some other 32-tools/bios may have overflow and unexpected behavior.